java/src/com/here/flexpolyline/PolylineEncoderDecoderTest.java

```diff
@@ -193,6 +206,117 @@ private void testLatLngZDecode() {
         }
     }
 
+    private static final String TEST_FILES_RELATIVE_PATH = "./test/";
```

Comment:
Would you run the tests from the `flexible-polyline` folder? I am not sure about Java's conventions, but to me it makes more sense to run the tests from the `flexible-polyline/java` folder, in which case this would have to be `"../test/"`.

I might be wrong, please enlighten me if so :-)

Reply:
Yep previously i have referred the test files within java package. But it is good to use common test folder, to ensure single point of updates and test. This is suggested by Vetter. It still work for the new end user, since they fork the entire project into their system. So no issues on the path.

Reply:
I think I haven't be clear enough. The only way this test is going to run is if it is run from the root directory, `flexible-polyline`, with the command 
```
$ java -cp java/src com.here.flexpolyline.PolylineEncoderDecoderTest
       ^^^^^^^^^^^^
```
Otherwise it wil fail with a `NoSuchFileException`.
This is neither intuitive nor good practice. Please rewrite so that the test file path is not relative to the CWD, which can change anyway, but to the source file, `PolylineEncoderDecoderTest.java`, or with another solution so the tests can not only be run with the addition of a classpath.

Reply:
Thank you! Lene. I have made the proposed change of file path relative to the CWD.